src/download.md

```diff
@@ -21,10 +21,10 @@ Download the latest Apache Cassandra 3.11 release: {{ "3.11" | full_release_link
 
 The following older Cassandra releases are still supported:
 
-* Apache Cassandra 3.0 is supported until **6 months after 4.0 release (date TBD)**. The latest release is {{ "3.0" | full_release_link }}.
-* Apache Cassandra 2.2 is supported until **4.0 release (date TBD)**. The latest release is {{ "2.2" | full_release_link }}.
-* Apache Cassandra 2.1 is supported until **4.0 release (date TBD)** with **critical fixes only**. The latest release is
-  {{ "2.1" | full_release_link }}.
+* Apache Cassandra 3.11 will fully be supported until **April 30, 2022** and with **critical fixes only** until **April 30, 2023** The latest release is {{ "3.11" | full_release_link }}.
+* Apache Cassandra 3.0 will be supported until **April 30, 2022** with **critical fixes only**. The latest release is {{ "3.0" | full_release_link }}.
+* Apache Cassandra 2.2 will be supported until **April 30, 2021**. with **critical fixes only**. The latest release is {{ "2.2" | full_release_link }}.
+* Apache Cassandra 2.1 will be supported until **April 30, 2021** with **critical fixes only**. The latest release is {{ "2.1" | full_release_link }}.
```

Comment:
i wonder if we should say "maintained" instead of "supported". 

Technically speaking, the community does not provide support. We maintain those branches, and commercial companies provide support for them (and more if they wish to).

I'm not sure, and can't remember, if this has ever been discussed.

Reply:
+1 on "maintained"